packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -96,7 +96,7 @@ const SearchFacets: FC<SearchFacetsProps> = ({
   useEffect(() => {
     fetchAggregations(searchParameters);
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
should we remove the eslint disable here?

Reply:
nope the one here is necessary, otherwise `eslint` will automatically add `searchParameters` to the dependencies which breaks some functionalities

Reply:
I thought that the point of this PR was to remove the linter ignores?

Reply:
Yes, but in some cases like this one it’s actually necessary I believe. Since we don’t want to include the whole ‘searchParameters’ as dependency just one of its values